segments/weather.sh

```diff
@@ -11,7 +11,7 @@ TMUX_POWERLINE_SEG_WEATHER_UPDATE_PERIOD_DEFAULT="600"
 # 1. Go to Yahoo weather http://weather.yahoo.com/
 # 2. Find the weather for you location
 # 3. Copy the last numbers in that URL. e.g. "http://weather.yahoo.com/united-states/california/newport-beach-12796587/" has the numbers "12796587"
-TMUX_POWERLINE_SEG_WEATHER_LOCATION=""
+TMUX_POWERLINE_SEG_WEATHER_LOCATION_DEFAULT=""
```

Comment:
Since the default value is always empty string, it's pointless to have I would say. I think we can remove it all together.